draftlogs/7012_add.md

```diff
@@ -0,0 +1 @@
+ - Plots can now have subtitles [[#7012](https://github.com/plotly/plotly.js/pull/7012)]
```

Comment:
Please consider updating this description to be more precise something like "add `subtitle` to `layout.title`".

Reply:
Done